src/otel-collector/config.yaml

```diff
@@ -6,11 +6,15 @@ dist:
   output_path: otel-collector
   version: 0.6.0
 exporters:
+  - gomod: go.opentelemetry.io/collector/exporter/debugexporter v0.104.0
   - gomod: go.opentelemetry.io/collector/exporter/otlpexporter v0.104.0
   - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/exporter/fileexporter v0.104.0
   - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/exporter/prometheusexporter v0.104.0
   - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/exporter/prometheusremotewriteexporter v0.104.0
+extensions:
+- gomod: go.opentelemetry.io/collector/extension/zpagesextension v0.104.0
```

Comment:
nit: the rest of the YAML file indents the array items. Can you please align everything.